packages/bruno-app/src/utils/exporters/openapi-spec.js

```
@@ -361,6 +361,7 @@ export const exportApiSpec = ({ variables, items, name, environments }) => {
           case 'oauth2':
             if (!auth?.oauth2?.grantType) break;
             const { authorizationUrl, accessTokenUrl, callbackUrl, scope } = auth?.oauth2;
```

Biome (2.4.15)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)